### DISCUSSION TITLE
Which caldera version do i use

### DISCUSSION THREAD

**Discussion by benesch1** [mitre/caldera#2747] [Q&A] [answered]
Dear community,

how do i know which caldera version am i using. Someone else did install this instance and i would like to see which version it was.

Thanks in advance.

Best regards...

**Comment by clenk** [comment, marked as answer]
If you make a GET request to the `/api/v2/health/` API endpoint it will include the version information, but this only works if it's v4.0+.
If it's v3.1+, you can also look in the `caldera/app/version.py` file.
If you run `git rev-parse HEAD` in the caldera directory it will tell you the git commit hash. If you post it here I can tell you what version it's from.

**Reply by benesch1** [reply]
Thanks a lot
